recipe/build.sh

```diff
@@ -1,28 +1,24 @@
 #!/bin/bash
 
-export CFLAGS="-I$PREFIX/include -L$PREFIX/lib"
+set -e
 
-# As of Mac OS 10.8, X11 is no longer included by default
-# (See https://support.apple.com/en-us/HT201341 for the details).
-# Due to this change, we disable building X11 support for cairo on OS X by
-# default.
-export XWIN_ARGS=""
-if [ $(uname) == Darwin ]; then
-    export XWIN_ARGS="--disable-xlib -disable-xcb --disable-glitz"
-fi
+export CFLAGS="-I$PREFIX/include -L$PREFIX/lib"
 
-./configure \
-    --prefix="${PREFIX}" \
-    --enable-warnings \
-    --enable-ft \
-    --enable-ps \
-    --enable-pdf \
-    --enable-svg \
-    --disable-gtk-doc \
-    $XWIN_ARGS
+configure_args=(
+    --prefix="$PREFIX"
+    --enable-warnings
+    --enable-ft
+    --enable-ps
+    --enable-pdf
+    --enable-svg
+    --disable-gtk-doc
```

Comment:
I'm ambivalent, but have been less active here. If it helps you with your work here, then I'm +1.